app/docs/md/learn/practices/testing.md

```diff
@@ -85,3 +85,40 @@ test('Shut down local server', async t => {
 })
 ```
 
+## Component Testing
+
+If you like to run functional component tests, we recommend to use [WebdriverIO](https://webdriver.io/) as browser automation framework. The benefits of using an actual browser compared to JSDOM outweigh the minimal speed detriment that comes with spinning up a browser instance. Furthermore WebdriverIO is based on [WebDriver](https://www.w3.org/TR/webdriver/) as browser automation web standard which guarantees you interact with your component as close as possible as a user would do.
```

Comment:
A few small suggested adjustments:

```suggestion
If you'd like to run functional component tests, we recommend using [WebdriverIO](https://webdriver.io/) as a browser automation framework. The benefits of using an actual browser compared to JSDOM outweigh the minimal speed detriment that comes with spinning up a browser instance. Furthermore, WebdriverIO is based on [WebDriver](https://www.w3.org/TR/webdriver/), a browser automation web standard which guarantees that your tests interact with your component as closely as possible to how a user would.
```

Overall, though, I feel that this passage is lacking a bit of context. IMO the second sentence should be prefaced by explicitly mentioning that other similar testing frameworks use JSDOM (and at least a link to a page defining what JSDOM is), and that WebDriver uses an actual browser instead.

Reply:
Thanks for the suggestions, I will take another stab at this.